packages/tooltip/stories/tail.stories.js

```diff
@@ -0,0 +1,133 @@
+import { storiesOf } from '@open-wc/demoing-storybook';
+import { html } from '@lion/core';
+
+import '@lion/button/lion-button.js';
+import '../lion-tooltip.js';
+
+storiesOf('Tooltip', module).add('Tail', () => {
+  const bw = 6;
+  const bc = 'black';
+  const popperConfig = {
+    modifiers: {
+      keepTogether: {
+        enabled: true,
+      },
+      arrow: {
+        enabled: true,
+      },
+    },
+  };
+  const placements = [
+    'top-start',
+    'top',
+    'top-end',
+    'right-start',
+    'right',
+    'right-end',
+    'bottom-start',
+    'bottom',
+    'bottom-end',
+    'left-start',
+    'left',
+    'left-end',
+  ];
+  return html`
+    <style>
+      .wrapper {
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
+        column-gap: 16px;
+        row-gap: 16px;
+        width: 900px;
+        margin: 100px auto;
+      }
+
+      .box {
+        border: 1px solid silver;
+      }
+
+      .canvas {
+        display: flex;
+        flex-direction: column;
+
+        align-items: center;
+        justify-content: center;
+
+        height: 100px;
+
+        border-bottom: 1px solid silver;
+      }
+
+      .config {
+        display: block;
+        padding: 16px;
+      }
+
+      lion-button span {
+        padding: 16px 0;
+      }
+
+      .tooltip {
+        position: absolute;
+        background: black;
+        color: white;
+        padding: 16px;
+      }
+
+      .arrow {
+        position: absolute;
+        width: 0;
+        height: 0;
+
+        border-color: red;
+        border-style: solid;
+      }
+
+      [x-placement^='top'] .arrow {
+        border-width: ${bw}px ${bw}px 0 ${bw}px;
+        border-color: ${bc} transparent transparent transparent;
+        bottom: -${bw}px;
```

Comment:
I wonder if we should do this here, or simply compensate for the arrow height/width in the popperConfig offset modifier. Probably the latter if you ask me, but it's an implementation detail so don't worry about it too much for now :).

Reply:
^ was wrong about this 🙈